src/keys.ts

```diff
@@ -10,6 +10,7 @@ interface I18NextParsedOptions {
   contexts: string[] | boolean;
   hasCount: boolean;
   ns: string | null;
+  defaultValue?: string;
```

Comment:
Is there any good reason for making this value optional? I think it could be `string | null` unless we want to allow `null` default values (which I don't think is possible) in which case we could either provide a `NOT_SET` symbol or just use an explicit `undefined`. I find that optional values makes it hard to reason about data structures. I prefer to avoid them whenever possible.